pkg/queue/manager.go

```diff
@@ -329,12 +330,14 @@ func (m *Manager) AddOrUpdateWorkloadWithoutLock(w *kueue.Workload) bool {
 	qKey := workload.QueueKey(w)
 	q := m.localQueues[qKey]
 	if q == nil {
+		log.Printf("LocalQueue for workload not found; queueKey: %s, workload: %s", qKey, w.Name)
```

Comment:
I think it’s much better to return an error here with a message like 'LocalQueue for workload not found' or 'ClusterQueue for workload not found'.

Reply:
for sure :)